vmdb/app/models/ems_amazon.rb

```diff
@@ -125,6 +125,21 @@ def vm_reboot_guest(vm, options = {})
     $log.error "MIQ(#{self.class.name}##{__method__}) vm=[#{vm.name}], error: #{err}"
   end
 
+  def stack_create(stack_name, template, options = {})
+    stack = cloud_formation.stacks.create(stack_name, template.content, options)
```

Comment:
You do not need the `stack` variable here, just add `.stack_id` to the end of the line.